### ISSUE TITLE
Part BOM is not displayed #2

### ISSUE THREAD

**Issue by sintech** [inventree/InvenTree#1861]
Seems to be BOM issue again.
If it matters: I changed my deployment to gunicorn + apache mod_proxy.

# Version Information:
InvenTree-Version: 0.3.1
Django Version: 3.2.4
Commit Hash: bcfb3ac
Commit Date: 2021-07-24
Database: mysql
Debug-Mode: True
Deployed using Docker: False

![изображение](https://user-images.githubusercontent.com/10057327/126893689-a5612e54-1353-48bf-ac56-48850828e4e8.png)
![изображение](https://user-images.githubusercontent.com/10057327/126893704-a2a48fc4-c3d6-4f53-8571-48801d58d6e0.png)

**Comment by SchrodingersGat** [comment]
What page / view are you trying to access when you see this error?

**Comment by sintech** [comment]
Part page with id 489. 
When part BOM did not load I opened dev console and saw this error.

**Comment by eeintech** [comment]
Umm yeah there is something fishy happening in the `price_range` calculation:

``` json
        "purchase_price_avg": "7,393.1687 руб.",
        "purchase_price_max": "7393.168712",
        "purchase_price_min": "7393.168712",
        "purchase_price_range": "7,393.1687 руб.",
```

Given `7,393.1687 руб.` is not a valid `Decimal` format, it will hit that error you reported when fetching the view.
I'll take a look.

**Comment by eeintech** [comment]
So never mind I did not know `руб.` was the representation of the currency. I found a weird bug in the purchase price conversion which I believe was causing this issue. I fixed it in #1866 so once merged let me know if that does it

**Comment by sintech** [comment]
Thanks @eeintech, now it works great.
